src/main/resources/api.yaml

```diff
@@ -619,3 +655,20 @@ components:
           $ref: '#/components/schemas/AvailabilityDto'
       required:
         - userId
+
+    JiraSprintsDto:
+      type: object
+      properties:
+        projectId:
+          type: integer
+          format: int64
+        sprintId:
+          type: integer
+        sprintNames:
```

Comment:
Nie mam pojęcia, jak to się będzie identyfikować, ale zakładam, że użytkownik ma dostać nazwę?

Reply:
Liczyłam na to, że magiczna matka chrzestna się zjawi i mi powie, ale to chyba sama muszę poszukać :(